.python-version

```diff
@@ -1 +1 @@
-3.8.12
+3.8.10
```

Comment:
We had to downgrade Python here because 3.8.11 and 3.8.12 are not available for Windows.